src/status_im/data_store/realm/schemas/account/v21/message.cljs

```diff
@@ -0,0 +1,44 @@
+(ns status-im.data-store.realm.schemas.account.v21.message
+  (:require [taoensso.timbre :as log]
+            [clojure.string :as str]))
+
+(def schema {:name       :message
+             :primaryKey :message-id
+             :properties {:message-id     :string
+                          :from           :string
+                          :to             {:type     :string
+                                           :optional true}
+                          :group-id       {:type     :string
+                                           :optional true}
+                          :content        :string
+                          :content-type   :string
+                          :username       {:type     :string
+                                           :optional true}
+                          :timestamp      :int
+                          :chat-id        {:type    :string
+                                           :indexed true}
+                          :outgoing       :bool
+                          :retry-count    {:type    :int
+                                           :default 0}
+                          :message-type   {:type     :string
+                                           :optional true}
+                          :message-status {:type     :string
+                                           :optional true}
+                          :user-statuses  {:type       :list
+                                           :objectType :user-status}
+                          :clock-value    {:type    :int
+                                           :default 0}
+                          :show?          {:type    :bool
+                                           :default true}}})
```

Comment:
It's little bit hard to see from this, but what was changes in message schema between versions V19 -> V21 ?

Reply:
true, there is no need for this duplication. removed it in the last commit.